src/app/util/util.cpp

```diff
@@ -770,6 +783,11 @@ EmberStatus emberAfSendImmediateDefaultResponseWithCallback(EmberAfStatus status
 
 EmberStatus emberAfSendImmediateDefaultResponse(EmberAfStatus status)
 {
+    if (chip::app::Compatibility::IMEmberAfSendImmediateDefaultResponseHandle(status))
+    {
+        // If the compatibility can handle this response
+        return EMBER_SUCCESS;
+    }
```

Comment:
Maybe you should move this code to https://github.com/project-chip/connectedhomeip/blob/8c93ec6b67f9c37cfeec9fa1d2cb93266eb4aa95/src/app/util/util.cpp#L776 since there could be multiple codepaths that leads to `emberAfSendImmediateDefaultResponseWithCallback` ?

Reply:
Thanks!